src/report/script.R

```diff
@@ -12,6 +12,12 @@ esets_normalized <- lapply(par$eset, function(eset_path) {
   return(file.path(normalizePath(dirname(eset_path)), basename(eset_path)))
 })
 
+if(!is.null(par$preproc_params)){
+  preproc_params_normalized <- file.path(normalizePath(dirname(par$preproc_params)), basename(par$preproc_params))
+}else{
+  preproc_params_normalized <- NULL
+}
+
```

Comment:
I prefer this format, but that is just nitpicking 🤷 
```suggestion
preproc_params_normalized <- NULL
if(!is.null(par$preproc_params)){
  preproc_params_normalized <- file.path(normalizePath(dirname(par$preproc_params)), basename(par$preproc_params))
}

```